docs/install.md

```diff
@@ -13,7 +13,7 @@ You can download the 8.2 ISO here: <https://mirrors.xcp-ng.org/isos/8.2/xcp-ng-8
 SHA256 checksums, GPG signatures and net-install ISO are available [here](https://xcp-ng.org/#easy-to-install).
 
 
-Then, create the install media (e.g. USB key):
+Then, create the install media (e.g. a USB key 1GB or larger should work):
```

Comment:
Would it be "an USB key"?

Reply:
If you are referring to "a" vs "an" -- in proper English it would be "a".

Reply:
That's why I used "would" because I wasn't sure. If you say it's "a", then it is :+1: